pyartcd/pyartcd/pipelines/binary_release_konflux.py

```
@@ -0,0 +1,582 @@
+
+        stage_rpa = "n/a"
+        prod_rpa = "n/a"
+        try:
+            config_path = self.shipment_data_repo._directory / "config.yaml"
+            if config_path.exists():
+                with open(config_path, 'r') as f:
+                    shipment_config = stdlib_yaml.safe_load(f) or {}
+                app_env_config = shipment_config.get("applications", {}).get(application, {}).get("environments", {})
+                stage_rpa = app_env_config.get("stage", {}).get("releasePlan", "n/a")
+                prod_rpa = app_env_config.get("prod", {}).get("releasePlan", "n/a")
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)